src/python/grpcio/grpc/utilities.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2016, Google Inc.
```

Comment:
I'm not too hot on the utilities.py name.  This file seems to only contain helper methods for rpc handler construction, I would prefer to see it called handlers.py, rpc_handlers.py, or something along those lines.